vita3k/renderer/src/vulkan/screen_filters.cpp

```diff
@@ -44,7 +44,6 @@ SinglePassScreenFilter::~SinglePassScreenFilter() {
     vk::Device device = screen.state.device;
     // this will only happen when the user changes the option in the GUI, we can afford to waitIdle
     device.waitIdle();
-    vao.destroy();
     device.destroy(pipeline);
     device.destroy(pipeline_layout);
     device.freeDescriptorSets(descriptor_pool, descriptor_sets);
```

Comment:
```suggestion
```
Can you remove this line too? It is not necessary and causing a validation error.